### ISSUE TITLE
Swipe up to enter Fullscreen mode

### ISSUE THREAD

**Issue by pc00per** [TeamNewPipe/NewPipe#7624]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### What feature do you want?
Swipe up video to enter Full Screen (similar to official YouTube)

#### Why do you want this feature?

The only way to enter fullscreen mode is to tap tiny bottom right button, which is annoyingly pin-pointed way.

**Comment by SameenAhnaf** [comment]
An alternative already available. Go to `Settings`>`Video and Audio`>`Behavior`> Turn on `Start main player in full screen`

**Comment by pc00per** [comment]
> An alternative already available. Go to `Settings`>`Video and Audio`>`Behavior`> Turn on `Start main player in full screen`

True. I did consider this alternative though. But I thought that a gesture will make the experience seemless experience.
Anyways, I'll wait for authors if they could implement the gesture, as it could depend on the complexity of implementing it.

**Comment by binex-dsk** [comment]
There should be a way to enter fullscreen while not in fullscreen. So I can switch the two at will, but I don't need rotation on

**Comment by github-account1111** [comment]
Also swipe down to exit fullscreen.

The button is annoying and slow to use, but also the gyroscope is absolutely dismal on most Android phones.
